executable-spec/src/Cardano/Ledger/Spec/STS/Update/Data.hs

```diff
@@ -130,8 +130,8 @@ vpDurationToSlotCnt :: VPDuration -> SlotCount
 vpDurationToSlotCnt  d =
   case d of
     VPMin -> SlotCount 20
-    VPMedium -> SlotCount 50
-    VPLarge -> SlotCount 100
+    VPMedium -> SlotCount 40
```

Comment:
I realize now that these requiring 40 or 80 slots for the voting period is a lot. In the tests we want to make these smaller (0-20, 20 for the large voting period for instance), so we should find a mechanism to make the translation of voting period duration to slot count configurable.

This would be a separate issue, but I'd propose to eliminate the `VPDuration` and specify this simply in slots.

Reply:
OK. TODO comment added